powerline.el

```diff
@@ -499,23 +499,23 @@ static char * %s[] = {
    'mouse-face 'mode-line-highlight
    'local-map (make-mode-line-mouse-map
                'mouse-1 (lambda () (interactive)
-                          (setq powerline-buffer-size-suffix
-                                (not powerline-buffer-size-suffix))
-                          (force-mode-line-update)))))
+                                (setq powerline-buffer-size-suffix
+                                  (not powerline-buffer-size-suffix))
+                                (force-mode-line-update)))))
 
 ;;;###autoload (autoload 'powerline-buffer-id "powerline")
 (defun powerline-buffer-id (&optional face pad)
   (powerline-raw
    (format-mode-line
     (concat " " (propertize
-		 (format-mode-line mode-line-buffer-identification)
-		 'face face
-		 'mouse-face 'mode-line-highlight
-		 'help-echo "Buffer name\n\ mouse-1: Previous buffer\n\ mouse-3: Next buffer"
-		 'local-map (let ((map (make-sparse-keymap)))
-			      (define-key map [mode-line mouse-1] 'mode-line-previous-buffer)
-			      (define-key map [mode-line mouse-3] 'mode-line-next-buffer)
-			      map))))
+                 (format-mode-line mode-line-buffer-identification)
+                 'face face
+                 'mouse-face 'mode-line-highlight
+                 'help-echo "Buffer name\n\ mouse-1: Previous buffer\n\ mouse-3: Next buffer"
+                 'local-map (let ((map (make-sparse-keymap)))
+                              (define-key map [mode-line mouse-1] 'mode-line-previous-buffer)
+                              (define-key map [mode-line mouse-3] 'mode-line-next-buffer)
+                              map))))
```

Comment:
please don't make all the indentation changes. i don't think mine were incorrect but i don't want to mess up the history

Reply:
Why not fix inconsistency once and for all as a separate commit?  If those tabs would have been used everywhere, I wouldn't even bother as that would have been consistent, which it was not.

Reply:
ok, fair enough , i pushed indentation changes in as separate commit. you'll need to rebase.

Reply:
Cool!  Rebased, check it out please. 👍